src/behaviors/ElementStatusBehavior.php

```diff
@@ -1,20 +1,16 @@
 <?php
+
 namespace putyourlightson\elementstatusevents\behaviors;
 
 use Craft;
 use craft\base\Element;
+use putyourlightson\elementstatusevents\ElementStatusChange;
+use putyourlightson\elementstatusevents\events\StatusChangeEvent;
 use yii\base\Behavior;
+use yii\base\Event;
 
 class ElementStatusBehavior extends Behavior
 {
-    // Constants
-    // =========================================================================
-
-    /**
-     * @event Event
-     */
-    const EVENT_STATUS_CHANGED = 'statusChanged';
```

Comment:
> Why moving the event name from the Behavior to the main class?

The event name is the "interface" for the consumer of the library. The Behavior is an implementation detail he/she should not worry about. And, decoupling event from the Behavior allows us to fire the event independently. E.g. in 
`commands/ScheduledElements::fireEvent()`
https://github.com/putyourlightson/craft-element-status-events/pull/1/files#diff-0068c7f8d063b0e3caf057549ad8a6d9R113